src/blocks/homepage-articles/class-wp-rest-newspack-articles-controller.php

```diff
@@ -80,6 +80,7 @@ public function get_items( $request ) {
 		$next_url = '';
 
 		Newspack_Blocks::filter_excerpt_length( $attributes );
+		Newspack_Blocks::filter_excerpt_ellipses( $attributes );
```

Comment:
Looks like this doesn't match the method name in `class-newspack-blocks.php`? This works for me:

```suggestion
		Newspack_Blocks::filter_excerpt_more( $attributes );
```